backend/drm/drm.c

```diff
@@ -598,6 +598,8 @@ static bool wlr_drm_output_set_cursor(struct wlr_output *_output,
 		wlr_matrix_texture(plane->matrix, plane->width, plane->height,
 			output->output.transform ^ WL_OUTPUT_TRANSFORM_FLIPPED_180);
 
+		// TODO the image needs to be rotated depending on the output rotation
```

Comment:
This can be done by translating [-hotspot, -hotspot], rotating, translating [+hotspot, +hotspot], then translating to the final place on screen.